nikola/plugins/template/mako.py

```diff
@@ -61,7 +61,11 @@ def get_string_deps(self, text, filename=None):
         for n in lex.template.nodes:
             keyword = getattr(n, 'keyword', None)
             if keyword in ["inherit", "namespace"] or isinstance(n, parsetree.IncludeTag):
-                deps.append(n.attributes['file'])
+                if n.attributes["file"] == n.parsed_attributes["file"].strip("'"):
+                    deps.append(n.attributes['file'])
+                else:
+                    LOGGER.warning("Ignoring file=%s for template string dependencies in %s",
```

Comment:
Perhaps we could add a special-case for the comment system case to avoid dependency glitches and warnings?

Reply:
Good idea, I put a test that it's the comments_helper include and removed the warning